src/index.ts

```diff
@@ -39,7 +39,29 @@ export function start() {
     }
     disposables.push(theia.commands.registerCommand(quickPickTestObjCommand, (...args: any[]) => testQuickPickObject()));
 
+    const informationMessageTestCommand = {
+        id: 'simple-plugin-information-message-command',
```

Comment:
could you rename the id by using prefix : 'frontend-plugin' instead of 'simple-plugin'

Reply:
done